src/store/facade/DataTransfer.java

```diff
@@ -32,9 +32,8 @@
 	private int productReorderLevel;
 	private int orderQuantity;
 	private String orderId;
-	private String list;
```

Comment:
Removed -list: String

Reply:
This was originally meant for returning toString() Strings, if we were gonna use them instead of list of results. But Brahma didn't want that. He wanted us to have more control than just getting a long String and printing it...